test/e2e/self_hosted_test.go

```diff
@@ -35,15 +36,32 @@ var _ = Describe("When testing Cluster API working on self-hosted clusters", fun
 	})
 })
 
-var _ = Describe("When testing Cluster API working on self-hosted clusters using ClusterClass [ClusterClass]", func() {
+var _ = Describe("When testing Cluster API working on self-hosted clusters using ClusterClass and single node ControlPlane [ClusterClass]", func() {
```

Comment:
```suggestion
var _ = Describe("When testing Cluster API working on self-hosted clusters using ClusterClass [ClusterClass]", func() {
```
just to align to the upgrade test. we didn't mention single node specifically there. I'm fine if we want to mention it, but let's then do the same for `When upgrading a workload cluster using ClusterClass [ClusterClass]`